### PR TITLE
[RW-876][RISK=NO] Update homepage to match new styling

### PR DESCRIPTION
**Pull request by NehaBroad**
Replace this this template with your PR description.
Please remember to keep in mind the security levels outlined in [CONTRIBUTING.md](CONTRIBUTING.md) and to 
include a risk tag of the form `[risk=no|low|moderate|severe]` in the PR title

* **no**: None 
* **low**: Low chance of potential impact to, or exposure of patient data
* **moderate**: Moderate chance of potential impact to, or exposure of patient data
* **severe**: Severe chance of potential impact to, or exposure of patient data

Please also:

* Get thumbs from reviewer(s)
* Verify all tests go green, including CI tests

### CODE REVIEW

**Comment by blrubenstein** [comment]
Mind uploading some screenshots when you get the chance?

File: ui/src/app/views/homepage/component.html [hunk 1]
```diff
@@ -1,80 +1,135 @@
...
+
+                    <div class="col analyzeSingleHeader">Analyze and Publish With Workspaces</div>
+                    <div class="col">
+                        <div class="row linkGroup">
+                            <div class="col">
+                                <a class="links" [routerLink]="" (click)="addWorkspace()">Create a
+                                    workspace</a>
+                            </div>
+                            <div class="col">
+                                <a class="links" [routerLink]="">See All Notebooks</a>
```
Comment (blrubenstein) [inline ui/src/app/views/homepage/component.html]: Can we put a disabled on this or something? Or some sort of messaging. 

Also, I assume you talked with Lou and Fritz on this, but I am very hesitant to put in any code and pages where we don't have functionality or views yet.

File: ui/src/app/views/homepage/component.html [hunk 2]
```diff
@@ -1,80 +1,135 @@
...
+                <div class="row">
+                    <div class="col">
+                        <img src='/assets/icons/Analyze.svg' class="iconImage">
+                    </div>
+
+                    <div class="col analyzeSingleHeader">Analyze and Publish With Workspaces</div>
+                    <div class="col">
+                        <div class="row linkGroup">
+                            <div class="col">
+                                <a class="links" [routerLink]="" (click)="addWorkspace()">Create a
```
Comment (blrubenstein) [inline ui/src/app/views/homepage/component.html]: Don't need routerLink here.

File: ui/src/app/views/homepage/component.html [hunk 3]
```diff
@@ -1,80 +1,135 @@
...
+                                    workspace</a>
+                            </div>
+                            <div class="col">
+                                <a class="links" [routerLink]="">See All Notebooks</a>
+                            </div>
+                        </div>
+                        <div class="row">
+                            <div class="col">
+                                <a class="links" [routerLink]="" (click)="listWorkspaces()">See All
```
Comment (blrubenstein) [inline ui/src/app/views/homepage/component.html]: Similarly don't need a routerLink field here.

**Comment by NehaBroad** [comment]
Existing User :
screen shot 2018-07-06 at 2 20 58 pm

New User :
new user

Footer
footer

File: ui/src/app/views/homepage/component.css [hunk 4]
```diff
@@ -1,251 +1,221 @@
...
+    height:100%
 }
 
-.tip-subtitle {
-    padding-top: 3.7rem;
-    padding-left: 2.1rem;
-    font-size: .9rem;
-    line-height: 1rem;
+.card {
+    border-radius: 5px;
```
Comment (blrubenstein) [inline ui/src/app/views/homepage/component.css]: We should have a border radius for cards in the card master style file. Does this need to be here/is this different than that??

File: ui/src/app/views/homepage/component.css [hunk 5]
```diff
@@ -1,251 +1,221 @@
...
-    border-radius: .3125rem;
-    background-color: #60AEE0;
-    position: absolute;
-    color: #FFFFFF;
-    margin-top: 9rem;
-    margin-left: 2rem;
-    font-weight: 300;
-    font-size: .6rem;
-    cursor: pointer;
+.left {
```
Comment (blrubenstein) [inline ui/src/app/views/homepage/component.css]: If this is all these CSS classes contain, I might actually rather have them as inline styles? I'm not sure the class adds much.

File: ui/src/app/views/homepage/component.css [hunk 6]
```diff
@@ -1,251 +1,221 @@
...
-    padding-top: 1.2rem;
-    padding-left: 1.2rem;
-    padding-right: .2rem;
+
+.single {
+    height: 349px;
+    width: 1118px;
+    border-radius: 5px;
+    background-color: rgba(255,255,255,0.15);
+    box-shadow: 0 0 2px 0 rgba(0,0,0,0.12), 0 3px 2px 0 rgba(0,0,0,0.12);
```
Comment (blrubenstein) [inline ui/src/app/views/homepage/component.css]: This has a lot similar to `.double`. Can we abstract that to the card class? Or just do a shared class and then only have width different?

File: ui/src/app/views/homepage/component.css [hunk 7]
```diff
@@ -1,251 +1,221 @@
...
+    margin-left: 25%;
 }
 
-.profile-update-button:hover {
-    background-color: #4356a7;
+.learnHeading {
+    margin-left: 8%;
+}
+.analyzeHeading {
+    height: 68px;	width: 65%;
```
Comment (blrubenstein) [inline ui/src/app/views/homepage/component.css]: Nit: Newlines

File: ui/src/app/views/homepage/component.css [hunk 8]
```diff
@@ -1,251 +1,221 @@
...
-    border-bottom-width: 0;
-    border-right-width: 0;
-    border-left-width: 0;
-    width: 100%;
-    display: flex;
-    background-color: #483F4B;
-    height: 5rem;
-    position: absolute;
+.footerText .title {
+    color:white;
```
Comment (blrubenstein) [inline ui/src/app/views/homepage/component.css]: Nit: Spacing

File: ui/src/app/views/homepage/component.css [hunk 9]
```diff
@@ -1,251 +1,221 @@
...
 }
 
-.logo {
-    top: 1rem;
-    left: 3.25rem;
-    height: 3.5rem;
-    width: 7rem;
-    position: relative
+.linkGroup {
+    margin-top:16px;
```
Comment (blrubenstein) [inline ui/src/app/views/homepage/component.css]: Nit: Spacing

File: ui/src/app/views/homepage/component.css [hunk 10]
```diff
@@ -1,251 +1,221 @@
...
+
+.card-text {
+    height: 56px;
+    width: 284px;
+    color: #216FB4;
+    font-size: 20px;
+    font-weight: 500;
+    line-height: 28px;
+    margin-top: 20%;
+    margin-left:6%;
```
Comment (blrubenstein) [inline ui/src/app/views/homepage/component.css]: Nit: Spacing

File: ui/src/app/views/homepage/component.css [hunk 11]
```diff
@@ -1,251 +1,221 @@
...
+
```
Comment (blrubenstein) [inline ui/src/app/views/homepage/component.css]: Nit: Remove extra line

File: ui/src/app/views/homepage/component.css [hunk 12]
```diff
@@ -1,251 +1,221 @@
...
+}
+
+.plus-circle {
+    width: 32px;
+    height:32px;
```
Comment (blrubenstein) [inline ui/src/app/views/homepage/component.css]: Nit: Spacing

File: ui/src/app/views/homepage/component.html [hunk 13]
```diff
@@ -1,80 +1,135 @@
...
+                        <img class="icon analyzeIcon" src="/assets/icons/Analyze.svg">
+                    </div>
+                    <div class="row analyzeHeading">Analyze and Publish with Workspaces
+                    </div>
+                    <div class="row">
+                        <label class="text">Dolor sit amet consectetuer adipiscing sed diam
+                            euismod tincidunt ut laoreet dolore.</label>
+                    </div>
+                    <div class="row heading">
+                        <button class="btn btn-homepage" (click)="addWorkspace()">CREATE WORKSPACE</button>
```
Comment (blrubenstein) [inline ui/src/app/views/homepage/component.html]: If we can, I would have the text here be regularly cased (E.g., 'Create Workspace') and then do a `text-transform: uppercase` on it in styling.

File: ui/src/app/views/homepage/component.html [hunk 14]
```diff
@@ -1,80 +1,135 @@
...
         </div>
     </div>
+    <div *ngIf="!firstTimeUser">
+        <div class="row">
+            <div class="card single">
+                <div class="row">
+                    <div class="col">
+                        <img src='/assets/icons/Analyze.svg' class="iconImage">
+                    </div>
+
```
Comment (blrubenstein) [inline ui/src/app/views/homepage/component.html]: Nit: Extra line

File: ui/src/app/views/homepage/component.html [hunk 15]
```diff
@@ -1,80 +1,135 @@
...
-        Terms of Service
+    <div class="footer">
+        <div class="row">
+            <div class="footerTitle">How to Use the AoU Researcher Workbench</div>
+        </div>
+        <div class="row textGroup">
+            <div class="footerText">
+                <div class="title">Working Within Researcher Workbench</div>
+                <ul class="bullet">
+                    <li>Researcher Workbench Mission</li>
```
Comment (blrubenstein) [inline ui/src/app/views/homepage/component.html]: Similarly, I would color these grey, and make them disabled. Currently they look like links, and I think that is not what we want.

File: ui/src/app/views/homepage/component.css [hunk 16]
```diff
@@ -1,251 +1,233 @@
...
+.card-text {
+    height: 56px;
+    width: 284px;
+    color: #216FB4;
+    font-size: 20px;
+    font-weight: 500;
+    line-height: 28px;
+    margin-top: 20%;
+    margin-left: 6%;
+
```
Comment (UrsaStutsman) [inline ui/src/app/views/homepage/component.css]: small nit: extra line

File: ui/src/app/views/homepage/component.css [hunk 17]
```diff
@@ -1,251 +1,233 @@
...
+}
+
+.bullets {
+    color: #83C3EC;
+}
+
+.footerLink {
+    margin-left: 53rem;
+    color: #FFFFFF;
+}
```
Comment (UrsaStutsman) [inline ui/src/app/views/homepage/component.css]: again small nit, this time add a line

**Comment by calbach** [comment]
I like the new homepage. I also noticed there is a pretty glaring issue of the left and right margins on this page. I tried to fix this quickly but turns out the styling on other pages sort of depends on those margins. Are you already planning to fix this?

**Comment by NehaBroad** [comment]
@calbach yes  I will be taking care of this and creating a new PR for the same along with RW-854.

**Pull request merged**